src/main/scala/com/datastax/sparkstress/WriteTask.scala

```diff
@@ -274,7 +276,33 @@ class WriteWideRowByPartition(config: Config, ss: SparkSession) extends
   }
 
   def getDataFrame: org.apache.spark.sql.DataFrame = {
-    RowGenerator.getWideRowByPartitionDataFrame(ss, config.seed, config.numPartitions, config.totalOps, config.numTotalKeys)
+    RowGenerator.getWideRowByPartitionDataFrame(ss, config.seed, config.numPartitions, config.totalOps, config.numTotalKeys
+    )
   }
+}
+
+@WriteTest
+class CopyTable(config: Config, ss: SparkSession) extends
+  WriteTask[PerfRowClass](config, ss)(implicitly[RowWriterFactory[PerfRowClass]]) {
 
+  override def getTableCql(tbName: String): Seq[String] =
+    Seq(
+      s"""CREATE TABLE IF NOT EXISTS ${tbName}_copy
+       |(store text,
+       |order_time timestamp,
+       |order_number uuid,
+       |color text,
+       |size text,
+       |qty int,
+       |PRIMARY KEY (store, order_time, order_number))
+     """.stripMargin)
```

Comment:
Instead of hard-coding this here and duplicating what we're doing in `WritePerfRow`, I think it would be nice to have a generalized data model generator that takes as input the type of data model (e.g. perf, widerow, shortrow, etc.) and outputs the appropriate schema. With this new generator, all the data models would have the same columns we currently use for perfrow, with the primary key structured appropriately to our needs. This is a nice-to-have, that I'm okay doing in a follow-up PR if that's more productive.

Reply:
Yeah i think "generic data model thing" is something we should do in a a future pr.